src/aiff.rs

```diff
@@ -0,0 +1,117 @@
+use safe_transmute::{transmute_one, transmute_one_to_bytes};
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::io::SeekFrom;
+use std::io::Cursor;
+use crate::{Tag, Version};
+
+pub fn load_aiff_id3(path: &str) -> Result<Option<Tag>, Box<dyn std::error::Error>> {
+    let mut file = File::open(path)?;
+    loop {
+        //Read chunk ID
+        let mut chunk_id_raw: [u8; 4] = [0; 4];
+        //EOF
+        if file.read(&mut chunk_id_raw)? == 0 {
+            break;
+        }
+        let chunk_id = String::from_utf8_lossy(&chunk_id_raw);
+
+        //Read chunk size
+        let mut chunk_size_raw: [u8; 4] = [0; 4];
+        file.read(&mut chunk_size_raw).ok();
```

Comment:
All of these calls to `read` aren't good, because it's not guaranteed that the whole buffer was filled. [read_exact](https://doc.rust-lang.org/1.48.0/std/io/trait.Read.html#method.read_exact) is better in these situations.